lib/components/Editor/CustomExtensions/FixedMenu/EmojiOption.js

```diff
@@ -0,0 +1,40 @@
+import React, { useRef } from "react";
+import { Smiley } from "@bigbinary/neeto-icons";
+import Dropdown from "components/Common/Dropdown";
+
+import {
+  ICON_COLOR_ACTIVE,
+  ICON_COLOR_INACTIVE,
+  MENU_ICON_SIZE,
+} from "./constants";
+import EmojiPickerMenu from "../Emoji/EmojiPicker/EmojiPickerMenu";
+
+const EmojiOption = ({ editor }) => {
+  const dropdownRef = useRef();
+
+  return (
+    <Dropdown
+      ref={dropdownRef}
+      closeOnSelect={false}
+      customTarget={() => (
+        <button className="h-full p-3 transition-colors editor-fixed-menu--item">
+          <Smiley
+            size={MENU_ICON_SIZE}
+            color={
+              dropdownRef?.current?.visible
+                ? ICON_COLOR_ACTIVE
+                : ICON_COLOR_INACTIVE
+            }
+          />
+        </button>
+      )}
+      position="bottom-start"
+    >
+      <div className="overflow-hidden h-80">
+        <EmojiPickerMenu editor={editor} range={{}} />
```

Comment:
What is this range for? why is it empty?

Reply:
The `EmojiPickerMenu` uses `range` to remove the decorator (::). Since there is no decorator involved while using via dropdown, the range is empty. Instead, I shall specify`range={{}}` as default in the component itself to avoid this confusion.